src/NetworkMap/Certificates/View.elm

```diff
@@ -30,12 +31,36 @@ import NetworkMap.Certificates.Msg exposing (Msg(..))
 import Palette exposing (fillWidth, limitLayoutWidth, shortHashRaw)
 import Screen.Model as Screen
 import Time
+import Utils.ArrayExtras as ArrayExtras
+
+
+flatMaybes : Array (Maybe a) -> Array a
+flatMaybes ar =
+    ar
+        |> A.foldr
+            (\m ->
+                \a ->
+                    case m of
+                        Just el ->
+                            A.append a (A.fromList [ el ])
+
+                        Nothing ->
+                            a
+            )
+            A.empty
 
 
 view : Screen.Model -> Model -> Element Msg
 view screen networkModel =
+    let
+        maybes =
+            networkModel.certificates |> A.map (\{ trustIds } -> trustIds |> A.map (\p -> Dict.get p networkModel.trusts))
```

Comment:
Have you applied the formatter? Lines are pretty long in this PR.

Reply:
Yes, I applied. I'll recheck it.